### ISSUE TITLE
"Show only packages with updates" box not always in sync with actual behavior

### ISSUE THREAD

**Issue by RiseT** [chocolatey/ChocolateyGUI#328]
Windows 10 x64, Chocolatey 0.10.0, ChocolateyGUI 0.13.2.0

**Steps to reproduce:**

1. Enable "Show only packages with updates"
2. Click "Update all" (of course only works if there is at least one update available)
3. Wait until updating has been completed

**Current result:**
While "Show only packages with updates" is **still ticked**, i. e. enabled, **all installed packages** are shown. Even after clicking "Refresh packages", this doesn't change. All installed packages shown, "Show only packages with updates" still ticked.
You have to untick and re-tick "Show only packages with updates" to actually only display the packages with updates.

**Expected result:**

- **Display only the remaining (if any) packages with updates** after updating has been processed. If there are none, display the empty list.

*OR*

- Display all installed packages, but **automatically untick** "Show only packages with updates"

**In other words: Make sure that the actual behavior and the status of the "Show only packages with updates" checkbox stay always in sync.**

**Comment by RiseT** [comment]
Duplicate of #288 
Closing.
